src/emulator/modules/SceAppUtil/SceAppUtil.cpp

```diff
@@ -273,9 +273,15 @@ EXPORT(int, sceAppUtilStoreBrowse) {
 }
 
 EXPORT(int, sceAppUtilSystemParamGetInt, unsigned int paramId, int *value) {
+    auto sys_lang = static_cast<SceSystemParamLang>(host.cfg.sys_lang);
```

Comment:
```suggestion
    const SceSystemParamLang sys_lang = static_cast<SceSystemParamLang>(host.cfg.sys_lang);
```

Reply:
const value can't works, i have already tested const auto, it require value editable
![image](https://user-images.githubusercontent.com/5261759/57078205-1ad9ac00-6cee-11e9-8581-43496b033696.png)

Reply:
Okayy, i forgot. Sorry :)